### ISSUE TITLE
Support TypeScript

### ISSUE THREAD

**Issue by danilobuerger** [mientjan/react-native-markdown-renderer#26]
As the title says, feature request for adding typings.

**Comment by mientjan** [comment]
Like a definition file? Sure ill add it

**Comment by mientjan** [comment]
So i finished the d.ts file for this library, its just a bit hard for me to test it at the moment. Can you check it with your react-native project @danilobuerger ?

https://github.com/mientjan/react-native-markdown-renderer/tree/feature/typescript-typings

**Comment by danilobuerger** [comment]
I didn't get it to work yet, but a few things:

Instead of 
```
import React, { Component } from "react";
```
shouldn't it be
```
import * as React from "react";
```
?

Instead of 
```
/// <reference types="markdownIt" />
```
shouldn't it be
```
/// <reference types="markdown-it" />
```
?

**Comment by danilobuerger** [comment]
Also 
```
renderer: (any) => View,
```
should have a parameter name like
```
renderer: (foo: any) => View,
```

**Comment by mientjan** [comment]
this is now released under v3.2.0

**Comment by danilobuerger** [comment]
Thanks!
